spaces/S000164/properties/P000078.md

```diff
@@ -0,0 +1,6 @@
+---
+space: S000164
+property: P000078
+value: true
+---
+
```

Comment:
Looks weird on the browser page if there is no text here and no reference.  Maybe add "By construction" for example.